pyomo/dataportal/plugins/sheet.py

```diff
@@ -54,7 +54,7 @@ def open(self):
         else:
             try:
                 self.sheet = ExcelSpreadsheet(self.filename, ctype=self.ctype)
-            except pyutilib.common.ApplicationError:
+            except ApplicationError:
```

Comment:
This is a problem: `ExcelSpreadsheet` is still going to raise a *pyutilib* `ApplicationError`, which will *not* be caught by a *pyomo* `ApplicationError`.

Reply:
Ew. You're right. This is not meant to do anything to `pyutilib.excel.spreadsheet` so I will revert it for now.